sw/device/tests/crt_test.c

```diff
@@ -23,6 +23,10 @@ extern char _data_start;
 extern char _data_end;
 extern char _data_init_start;
 
+// Symbols in the CRT assembly library.
+extern void crt_section_clear(void *start, void *end);
+extern void crt_section_copy(void *start, void *end, void *source);
```

Comment:
Nit: I would move these closer to the function that tests each.

Reply:
Done